yew-functional/src/lib.rs

```diff
@@ -308,6 +588,28 @@ where
     );
 }
 
+/// Sometimes, it's needed to manually define dependencies for [`use_effect`].
+/// In such cases, we use `use_effect_with_deps`.
+///
+/// # Example
+///
+/// ```rust
+/// use yew_functional::{function_component, use_effect_with_deps};
+/// use yew::prelude::*;
+///
+/// # #[function_component(UseEffectWithDeps)]
+/// # fn effect_with_deps() -> Html {
+///     use_effect_with_deps(
+///         move |_| {
+///             // ...
+///             || {}
+///         },
+///         (), // dependents
+///     );
+///
+/// #    html! {}
+/// # }
+/// ```
```

Comment:
This example isn't very useful, not even really as a test because it doesn't represent an actual use case.

Reply:
I'll remove it from the doc string but keep it in the website docs as it shows the signature of the Hook function.